### DISCUSSION TITLE
Would you please tell me the specific information of the online test error?

### DISCUSSION THREAD

**Discussion by gengdd** [PaddlePaddle/PaddleSpatial#125] [BaiduKDDCup2022] [answered]
Team name: Timeout
The result submitted today is a positive number online. After the background modification, I get -768, but I don't know what caused it. The code runs normally locally. Thanks.

**Comment by wang975679801** [comment]
I found the question in online test code. They evaluate the std of prediction/1000, if it is lower than 0.1, then it will be -768, which is very confusing. The accurate prediction of the second day is almost impossible, therefore the model will tend to predict the same value.  I think it is normal. I can't understant why it will be treated as abnormal prediction.

**Reply by gengdd** [reply]
I took a look at the updated code and see why, thanks！
Probably because the standard deviation of the 288 values predicted by the model is too small，and proportion of the same values is too large.
![image](https://user-images.githubusercontent.com/18337863/171030975-47a18365-2f1c-46e6-adf5-60c8c8e3e3d3.png)

**Comment by zhoujingbo** [comment, marked as answer]
> I took a look at the updated code and see why, thanks！ Probably because the standard deviation of the 288 values predicted by the model is too small，and proportion of the same values is too large. ![image](https://user-images.githubusercontent.com/18337863/171030975-47a18365-2f1c-46e6-adf5-60c8c8e3e3d3.png)

Thanks a lot, I think this is the point.

**Comment by ramber1836** [comment]
A more detailed error is to be returned rather than an invalid score. Please refer to the latest code and try it again.
